<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrdenProductos</class>
 <widget class="QDialog" name="OrdenProductos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resumen de Orden</string>
  </property>
  <widget class="QLabel" name="label_orden">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>14</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_cliente">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>36</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="butt_eliminar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>298</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total: L</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>298</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>14</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Orden:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>36</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cliente:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butt_send">
   <property name="geometry">
    <rect>
     <x>752</x>
     <y>348</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="butt_back">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>348</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_orden">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>58</y>
     <width>791</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>128</number>
   </attribute>
   <column>
    <property name="text">
     <string>Cantidad</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Descripcion</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Complemento 1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Complemento 2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Comentario</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Precio</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="butt_edit">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>290</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <zorder>label_orden</zorder>
  <zorder>label_cliente</zorder>
  <zorder>label_5</zorder>
  <zorder>label_total</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>butt_send</zorder>
  <zorder>butt_back</zorder>
  <zorder>butt_eliminar</zorder>
  <zorder>table_orden</zorder>
  <zorder>butt_edit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
